basic programm (#3)
raven-os/egg: README.md

# Egg

The installer of Raven-OS

> First you need to install gettext for the command msgfmt

Reviewer: ```suggestion

## Dependencies

Install the package `gettext` to use the command `msgfmt`. Depending on your distribution, it may be called `msgfmt`, `msgfmt3` or `msgfmt3.py`.
```

> First you need to install Python development for the command msgfmt3

Reviewer: This command doesn't look like it is available on other systems but Fedora/RHEL. Please use something else, or the user won't be able to build the project (which is a big no-no).

If, and only if, the license of `msgfmt3` permits you, you can redistribute it in this repository (given that the file isn't huge).

> At the racine of the repository launch:

Reviewer: ```suggestion
At the root of the repository, run:
```

> ``` find ./locales -name "*.po" | while read f; do msgfmt -o ${f%.po}.mo $f; done ``` 

Reviewer: ````suggestion
```bash
find ./locales -name "*.po" | while read f; do msgfmt -o ${f%.po}.mo $f; done
``` 

You may need to adapt this command depending on the alternative name `msgfmt` might have on your system.

## Running the intaller

To run Egg, type 

```bash
python3 ./egg.py
```

Note that only the root user is able to use Egg.
